### PR TITLE
New cookbook for OTEL

### PR DESCRIPTION
**Pull request by ornellaaltunyan**

### CODE REVIEW

File: examples/OTEL-logging/otel-logging.mdx [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+[OpenTelemetry](https://opentelemetry.io/docs/) (OTel) is an open-source observability framework designed to help developers collect, process, and export telemetry data from their applications for performance monitoring and debugging. It’s used by popular libraries like the [Vercel AI SDK](https://sdk.vercel.ai/), [LlamaIndex](https://docs.llamaindex.ai/en/stable/module_guides/observability/), and [Traceloop OpenLLMetry](https://www.traceloop.com/docs) for observability in AI applications. OTel support extends to many programming languages including Python, TypeScript, Java, and Go. 
+
+![Popular AI SDKs -> OTel -> traces](assets/otel.png)
+
+In Braintrust, we enable observability in AI applications through [**logging**](/docs/guides/logging). Logs are the recorded data and metadata from an AI routine — we record the inputs and outputs of your LLM calls to help you understand and debug your application. We want to make it simple for you to log to Braintrust from many different environments, so we defined a way to set up Braintrust as an OpenTelemetry backend. This guide will walk you through how to log to Braintrust from a sample project built with the Vercel AI SDK.
```
Comment (ankrgyl) [inline examples/OTEL-logging/otel-logging.mdx:7]: logging -> tracing?

**Review by ankrgyl — APPROVED** [review]

File: examples/OTEL-logging/otel-logging.mdx [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+Then, configure Next.js telemetry by adding the following to your `next.config.mjs` file:
+```
+nextConfig.experimental = {
+  instrumentationHook: true,
+};
+```
+
+You can then use the `experimental_telemetry` option to enable telemetry on supported AI SDK function calls. In your `route.ts` file, add this import statement:
+
+```tsx
```
Comment (aphinx) [inline examples/OTEL-logging/otel-logging.mdx]: I believe this import was already in the original so you can remove this

File: examples/OTEL-logging/otel-logging.mdx [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+          const trackingInformation = getTrackingInformation({ orderId });
+          await new Promise((resolve) => setTimeout(resolve, 500));
+          return trackingInformation;
+        },
+      },
+    },
+    experimental_telemetry: {
+      isEnabled: true,
+      functionId: "multi-step-tool-calls-demo",
+      metadata: { foo: "bar" },
```
Comment (aphinx) [inline examples/OTEL-logging/otel-logging.mdx:102]: up to you, but maybe you call out that you can add metadata here to make your LLM calls easier to track (and they'll end up as metadata in the Braintrust span)

**Review by aphinx — APPROVED** [review]

File: examples/OTEL-logging/otel-logging.mdx (hunk 2 above)
Comment (aphinx) [reply examples/OTEL-logging/otel-logging.mdx]: (I did change the line to remove an unused import but that was just OCD)

**Pull request merged**